libraries/rush-lib/src/logic/operations/ShellOperationRunnerPlugin.ts

```diff
@@ -100,11 +105,12 @@ function createShellOperations(
 function getScriptToRun(
   rushProject: RushConfigurationProject,
   commandToRun: string,
-  customParameterValues: ReadonlyArray<string>
+  customParameterValues: ReadonlyArray<string>,
+  shellCommand?: string
 ): string | undefined {
   const { scripts } = rushProject.packageJson;
 
-  const rawCommand: string | undefined | null = scripts?.[commandToRun];
+  const rawCommand: string | undefined | null = scripts?.[commandToRun] ?? shellCommand;
```

Comment:
If i understand the design correctly, should it be `shellCommand ?? scripts?.[commandToRun]`?